src/tools/cephfs/cephfs-shell

```diff
@@ -812,9 +812,16 @@ class CephFSShell(Cmd):
  
 if __name__ == '__main__':
     config_file = ''
-    if sys.argv[1] == '-c':
-        config_file = sys.argv[2]
+    main_parser = argparse.ArgumentParser(description = '')
```

Comment:
could you run `pep8` or `pycodestyle` against the new code, just to make sure we don't have more style warnings/errors ?

Reply:
@tchaikov Do I make this in a seperate pr?

Reply:
@rpavani1998 i don't think it's necessary. would be better to fix the
```
E251 unexpected spaces around keyword / parameter equals
```
in this very commit.

i just wanted to raise your attention here. as i am trying to get a pep8-clean cephfs-shell before it's packaged. and i am posting #23480, so probably you can (early) review it. i will test it after it's pep8 clean.